config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'customers/:id', to: 'customers#show', as: 'customer'
 
+  post 'customers/:id/add_item', to: 'customers#add_item', as: 'add_item_customer'
```

Comment:
To make this a restful route, we'd want to do something like `post "/customers/:id/items", to: "customer_items#create"` . We want to stick to this restful routing uri structure and route to the restful actions.